storage/onpremise/mongodb/variables.tf

```
@@ -1,36 +1,44 @@
-# Global variables
 variable "namespace" {
   description = "Namespace of ArmoniK resources"
   type        = string
+  default     = "default"
 }
 
-# Parameters for MongoDB
-variable "mongodb" {
-  description = "Parameters of MongoDB"
-  type = object({
-    image              = string
-    tag                = string
-    node_selector      = map(string)
-    image_pull_secrets = string
-    replicas_number    = number
-  })
+variable "labels" {
+  description = "Labels for the Kubernetes StatefulSet to be deployed"
+  type        = map(string)
+  default = {
+    "app"  = "storage"
+    "type" = "table"
+  }
 }
 
-variable "security_context" {
-  description = "security context for MongoDB pods"
+variable "name" {
+  description = "Name used for the helm chart release and the associated resources, must be shorter than 54 characters"
+  type        = string
+  default     = "mongodb-armonik"
+}
+
+variable "mongodb" {
+  description = "Parameters of the MongoDB deployment"
+
   type = object({
-    run_as_user = number
-    fs_group    = number
+    architecture          = optional(string, "replicaset") # "replicaset" or "standalone"
+    databases_names       = optional(list(string), ["database"])
+    helm_chart_repository = optional(string, "oci://registry-1.docker.io/bitnamicharts")
+    helm_chart_name       = optional(string, "mongodb")
+    helm_chart_version    = string
+    image                 = optional(string, "bitnami/mongodb")
+    image_pull_secrets    = optional(any, [""]) # can be a string or a list of strings
+    node_selector         = optional(map(string), {})
+    registry              = optional(string, "docker.io")
```

remove the default value to avoid the issue when using a private registry
 ```suggestion
    registry              = optional(string)
```